Cython/Debugger/libpython.py

```diff
@@ -628,7 +628,7 @@ class PyCFunctionObjectPtr(PyObjectPtr):
     _typename = 'PyCFunctionObject'
 
     def proxyval(self, visited):
-        m_ml = self.field('m_ml')  # m_ml is a (PyMethodDef*)
+        m_ml = self.field('m_ml') # m_ml is a (PyMethodDef*)
```

Comment:
All of the updates in this file are upstreamed from the Python3.7 branch of CPython